qasp/agile_bpa_qasp.md

```diff
@@ -0,0 +1,106 @@
+PERFORMANCE BASED QUALITY ASSURANCE SURVEILLANCE PLAN (QASP)
+=============================================================
+
+{{ project_name }}
```

Comment:
Do you have a preference? I suppose that camelcase will make javascript linters happy without making python/ruby linters unhappy, but the opposite isn't true, so camelcase may be the way to go.